### DISCUSSION TITLE
Destination NFS JSON

### DISCUSSION THREAD

**Discussion by bmanojlovic** [airbytehq/airbyte#36136] [New Connector Request]
### Connector Name

NFS JSON

### What type of integration

Destination

### Revelant Information

I have wrote this code so that connector can write to NFS share that is exposed to airbyte cluster
it is python code that  uses libnfs-python and libnfs user space implementation of NFS protocol 
[destination-nfs-json](https://github.com/bmanojlovic/destination-nfs-json) or in forked [airbyte](https://github.com/bmanojlovic/airbyte/) repo

It has same behavior as SFTP and Local JSON destinations.

### Contribute

- [X] Yes, I want to contribute

**Comment by marcosmarxm** [comment]
Why not extend the SFTP or local Json destination?

**Comment by bmanojlovic** [comment]
I am a bit confused how exactly you meant extend?
to make it possible to use same(similar) configuration or?
I mean of course it is possible to reuse parts of sftp (I did) but problem with local json is  that it is java based so not that easy to do it.

[a comment by an automated account is not shown]
